scripts/initiate_ritual.py

```diff
@@ -72,6 +75,14 @@
     help="The filepath of a file containing newline separated staking provider addresses.",
     type=click.File("r"),
 )
+@click.option(
+    "--autosign",
+    is_flag=True,
+)
```

Comment:
Does the --autosign parameter work for you? I ran the command on lynx with this and I still had to type the signer password every time.

Also,


```suggestion
@click.option(
    "--autosign",
    help="Allow the Ape account to automatically sign the transactions.",
    is_flag=True,
)
```

Reply:
Let's have a closer look in a follow-up PR / issue.